client/core/simnet_trade.go

```diff
@@ -146,20 +154,7 @@ func (s *simulationTest) waitALittleBit() {
 		s.client2.BaseWalletType == WTElectrum || s.client2.QuoteWalletType == WTElectrum {
 		sleep = 7 * time.Second
 	}
-	sleep *= sleepFactor
-	feeSleep := time.Millisecond * time.Duration(s.client1.dc().cfg.BroadcastTimeout/2)
-	if sleep < feeSleep {
-		// eth based assets need to wait for txn to be mined for fees
-		// to be populated here.
-		for bipID, _ := range s.client1.wallets {
-			if accountBIPs[bipID] {
-				sleep = feeSleep
-				s.client1.log.Infof("Waiting for paid order fees to be populated (%s).", sleep)
-				break
-			}
-		}
-	}
-	time.Sleep(sleep)
+	time.Sleep(sleep * sleepFactor)
```

Comment:
We don't need to wait really for this, just mine 2 blocks for inits, so this is moved.